e2e/tests/integration/arrivals-filter.ts

```diff
@@ -34,4 +34,93 @@ describe('Arrivals page filter', () => {
       .get('.danger > :nth-child(1)').contains("TS0123")
 
   });
+
+ //valid service types for filter ("J", "S", "Q", "G", "B", "R", "C", "L")
+
+  it('Filter flight by passenger flight only for ACL Forecast when LoadFactor is zero and valid service type', () => {
```

Comment:
we should be able to combine the test for the UI and the test for the export into the same test as E2E tests can be slow, it makes sense to test all the different cases with the same setup.